website/content/docs/agent/template.mdx

```diff
@@ -64,6 +64,16 @@ through this function will be based on the certificate's expiration.
 {{ pkiCert "pki/issue/my-domain-dot-com" "common_name=foo.example.com" }}
 ```
 
+To fetch the issuing CA for this mount, use:
+
+```
+{{- with secret "pki/cert/ca" -}}
+{{ "" | or ( .Data.certificate ) }}
```

Comment:
On an err  response from Vault though, would this avoid the error breaking the template?

Reply:
Not a panic, at least in `go-template`. If the template execution panics anywhere, then the template execution will be an error, though I don't know how that trickles up into `consul-template`.

Reply:
Hmmm yeah, I see what you mean. [This one with a map](https://go.dev/play/p/wIlbWxol8W6) does alright, but [with a struct](https://go.dev/play/p/CxNYpkjERvl) it panics. I think since Data is a `map` it won't panic then?

Reply:
Makes sense to me.

Reply:
@swenson Thanks! Updated.